drivers/input/stmpe811_base.c

```diff
@@ -302,6 +302,7 @@ STMPE811_HANDLE stmpe811_instantiate(FAR struct i2c_master_s *dev,
   ret = stmpe811_checkid(priv);
   if (ret < 0)
     {
+      nxmutex_destroy(&priv->lock);
```

Comment:
should be
```
#ifdef CONFIG_STMPE811_MULTIPLE
    g_stmpe811list = priv->flink;
#endif
    nxmutex_destroy(&priv->lock);
    kmm_free(priv);
```

Reply:
No, priv come from g_stmpe811 if CONFIG_STMPE811_MULTIPLE isn't defined.

Reply:
Yes. Thank you